tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -58,7 +63,10 @@ function SplashPage() {
 
   return (
     <>
-      <section className="usa-hero" aria-label="Introduction">
+      <section
+        className={`usa-hero usa-hero${randomIndex()}`}
```

Comment:
This will toggle through random indexes to iterate through the background pages on every render.